### DISCUSSION TITLE
Welcome to Simple File Manager for Amazon EFS Solution Discussions!

### DISCUSSION THREAD

**Discussion by dorrikh** [aws-solutions/simple-file-manager-for-amazon-efs#209] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by imdevopx** [comment]
Hi, 

is Simple File Manager for EFS can be done in Terraform?
